config/validated_config_db_connector.py

```diff
@@ -17,33 +19,79 @@ def __getattr__(self, name):
                 return self.validated_set_entry
             if name == "delete_table":
                 return self.validated_delete_table
+            if name == "mod_entry":
+                return self.validated_mod_entry
         return self.connector.__getattribute__(name)
 
     def make_path_value_jsonpatch_compatible(self, table, key, value):
+        def stringify_value():
+            nonlocal value
```

Comment:
>value [](http://example.com/codeflow?start=21&length=5)

Just move `value` to parameter list, so this function could be a normal function of the class.

Reply:
Changed function to be normal function of the class